### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by msha096** [hasanirtiza/Pedestron#68]
Dear author,

Thanks for your work.

Could you please tell me how did you train this model? RetinaNet with Guided Anchoring | CityPersons | ResNeXt | 11.7 | 41.5

I trained on the Citypersons dataset but the MR is only about 20% on Reasonable. What is your training strategy? Did you pre-trian your model on other datasets before training on the CP? And how about LR and epochs?

Thanks!!

**Comment by hasanirtiza** [comment]
For `RetinaNet with Guided Anchoring`, we pre-trained it on ECP dataset. We did not change the LR, You can train it for ~20-30 epochs on ECP and get the best model (in this case, best on CityPerson validation set) as a starting point for CityPerson. Usually this model will converge around 10-15 epochs on CityPerson.

**Comment by msha096** [comment]
> For `RetinaNet with Guided Anchoring`, we pre-trained it on ECP dataset. We did not change the LR, You can train it for ~20-30 epochs on ECP and get the best model (in this case, best on CityPerson validation set) as a starting point for CityPerson. Usually this model will converge around 10-15 epochs on CityPerson.

I would like to know the best MR it can achieve without pre-training on ECP, and I would like to only train it on the citypersons. I have trained 50 epochs, but the loss seems converge at around 10-20 epochs. The loss does not decrease significantly...keeps around 0.4.....

What should I do? BTW, how to change the learning rate when I reach a specific epoch number....? which file should I look at it?

Thanks for your help!!!

**Comment by hasanirtiza** [comment]
I do not remember exactly now the numbers. You can have an adaptive learning rate in `configs` I believe or in the training script. Check [mmdetection](https://github.com/open-mmlab/mmdetection) docs.
